package.json

```diff
@@ -4,24 +4,25 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "scripts": {
```

Comment:
For this section:

- added "prettier-check": "yarn prettier --check ./packages"
- abc'd the scripts to make them more readable (^ line changes, no changed functionality)

Reply:
> abc'd the scripts to make them more readable

Do we plan to enforce that or will that depend on people remembering to keep the order?

I don't know if I'd call that more readable myself. 😅 Before this change, the scripts were more or less in some logical groups (stuff for tests, then stuff related to Teleport Connect, then the Web UI, then some linting), now they're just in alphabetical order. To be honest I'd be for keeping them that way.